lib/msf/core/auxiliary/report.rb

```diff
@@ -12,6 +12,38 @@ module Auxiliary::Report
 
   optionally_include_metasploit_credential_creation
 
+  def create_cracked_credential(opts={})
+    begin
+      super(opts)
+    rescue NoMethodError => e
```

Comment:
How reliable is this?  Doesn't that mean a that a simple typo somewhere in or called by ```super``` could also result in this happening and it simply assuming the DB isn't functioning.  Smells like there should be a better way to check for this.